### ISSUE TITLE
Varargs notation `::Any...` is incorrectly flagged as unsused variable

### ISSUE THREAD

**Issue by lassepe** [julia-vscode/StaticLint.jl#226]
When using `::Any...` in a method definition, that argument is incorrectly flagged as "unused variable".

For example:

```julia
function my_function(::Any...)
    # ...
end
```

This warning is suppressed if braces are used:

```julia
function my_function(::(Any...))
    # ...
end
```

**Comment by ZacLN** [comment]
Fixed at https://github.com/julia-vscode/StaticLint.jl/pull/203/commits/8bb1572d92e27694c1c206b3a89337dbde591d4e
